src/features/home-page/homepage-nav-bottom/button-toggle-conditioner.js

```diff
@@ -1,29 +1,75 @@
-import React, { Component } from "react";
+import React, { Component } from 'react';
+import { firebase } from '../../../firebase';
 
 class ToggleConditioner extends Component {
   constructor(props) {
     super(props);
-    this.state = { isToggleOn: false };
+    this.state = {
+      turnOnOffConditioner: true,
+      isLoading: false,
+    };
     this.handleClick = this.handleClick.bind(this);
   }
 
-  handleClick() {
-    this.setState(prevState => ({
-      isToggleOn: !prevState.isToggleOn
-    }));
-  }
+  handleClick = () => {
+    const user = firebase.auth.currentUser;
+    this.setState({
+      isLoading: true,
+    });
+    if (user) {
+      firebase.db
+        .collection('users')
+        .doc(user.uid)
+        .collection('rooms')
+        .get()
+        .then(snapshot => {
+          snapshot.docs.forEach(document => {
+            const roomDocRef = firebase.db
+              .collection('users')
+              .doc(user.uid)
+              .collection('rooms')
+              .doc(document.id);
+            if (document.data().consistConditioner) {
+              roomDocRef
+                .collection('devices')
+                .get()
+                .then(snapshot => {
+                  snapshot.docs.forEach(document => {
+                    roomDocRef
+                      .collection('devices')
+                      .doc(document.id)
+                      .update({
+                        isOn: this.state.turnOnOffConditioner,
```

Comment:
here, every device with such field will be changed. Think about some unique name..